root/etc/cont-init.d/98-crowdsec

```diff
@@ -44,9 +44,10 @@ fi
 sed -i -r "s|API_KEY=.*$|API_KEY=${CROWDSEC_API_KEY}|" "${CONFIG_PATH}crowdsec-nginx-bouncer.conf"
 sed -i -r "s|API_URL=.*$|API_URL=${CROWDSEC_LAPI_URL}|" "${CONFIG_PATH}crowdsec-nginx-bouncer.conf"
 
+# Sed in CROWDSEC_MODE mode, if defined in the env, defaults to live
+sed -i -r "s|MODE=.*$|SECRET_KEY=${CROWDSEC_MODE-live}|" "${CONFIG_PATH}crowdsec-nginx-bouncer.conf"
+
 # Sed in ReCaptcha keys
-sed -i -r "s|SECRET_KEY=.*$|SECRET_KEY=${CROWDSEC_SECRET_KEY}|" "${CONFIG_PATH}crowdsec-nginx-bouncer.conf"
-sed -i -r "s|SITE_KEY=.*$|SITE_KEY=${CROWDSEC_SITE_KEY}|" "${CONFIG_PATH}crowdsec-nginx-bouncer.conf"
```

Comment:
Why are you removing the recaptcha support?

Reply:
maybe i messed up, let me review all that